### ISSUE TITLE
Adding functions for get_eigrp_config() and get_eigrp_neighbors()

### ISSUE THREAD

**Issue by WilliamMarti** [napalm-automation/napalm-base#183]
So today there are the following methods in napalm-base

`get_bgp_neighbors()`
`get_bgp_config()`

In the network I support we are heavy EIGRP users and could use similar functionality.  I think this comes back to a larger discussion from slack and [here](https://github.com/napalm-automation/napalm-base/issues/182), talking about vendor specific protocols.  So my proposal is to add:

`get_eigrp_neighbors()`
`get_eigrp_config()`

Realistically these would only ever be implemented in the Cisco napalm flavors.  

Looking at the `get_route_to` function, it should already support EIGRP routes.

**Comment by dbarrosop** [comment]
Yeah, I think those methods are fine. Is this something you are looking for to implement yourself?

**Comment by mirceaulinic** [comment]
Hi @WilliamMarti - are you still interested in this?

**Comment by WilliamMarti** [comment]
Apologies, totally fell off my radar.  

Yes I would like to implement this.  I will get working on the PR for napalm-base now.

**Comment by mirceaulinic** [comment]
@WilliamMarti no problem at all - just checking

**Comment by dbarrosop** [comment]
This looks good in principle, I would like to see an initial implementation though before approving and merging.
